components/src/List/List.story.js

```diff
@@ -0,0 +1,42 @@
+import React from "react";
+import { storiesOf } from "@storybook/react";
+import List from "./List";
+import ListItem from "./ListItem";
+
+storiesOf("List", module)
+  .add("List", () => (
+    <React.Fragment>
```

Comment:
Don't need `React.Fragment` if there is only one top-level node.

Reply:
Same for the stories below.

Reply:
fixed